### PR TITLE
added bottom navigation bar

### LINKED ISSUE #76: make bottom nav bar
The current app has 4 fragments , the new one needs to have a bottom nav(like youtube does) 
Which will have 5 fragments  namely: herald, issues, favourites, events  and utilities.
herald, utilities, events will open the recycler view that they currently do.
keep favourites and issues fragments empty for now.

### PR DESCRIPTION
**Pull request by Agrim1234**
fixes #76 
completed bottom bar with 5 fragments utilities, herald, favourites, issues, events

### CODE REVIEW

File: .idea/inspectionProfiles/Project_Default.xml [hunk 1]
```diff
@@ -1,362 +0,0 @@
-<component name="InspectionProjectProfileManager">
```
Comment (Rushi98) [inline .idea/inspectionProfiles/Project_Default.xml]: Don't delete this file

File: .idea/codeStyles/codeStyleConfig.xml [hunk 2]
```diff
@@ -1,5 +0,0 @@
-<component name="ProjectCodeStyleConfiguration">
-  <state>
```
Comment (Rushi98) [inline .idea/codeStyles/codeStyleConfig.xml]: This file is required for consistent code style

File: app/src/main/java/com/csatimes/dojma/activities/MainActivity.java [hunk 3]
```diff
@@ -432,11 +460,15 @@ private void setupViewPagers() {
...
         EventsFragment eventsFragmentFragments = new EventsFragment();
         Utilities utilitiesFragment = new Utilities();
+        IssuesFragment issuesFragment = new IssuesFragment();
+        favouritesfragment favouritesFragment = new favouritesfragment();
 
         eventsFragmentFragments.setOnTitleUpdateListener(this);
 
         adapter.addFragment(heraldFragmentFragment, "Herald", MAIN_ACTIVITY_HERALD_POS);
-        adapter.addFragment(gazettesFragment, "Gazettes", MAIN_ACTIVITY_GAZETTES_POS);
+        adapter.addFragment(gazettesFragment, "Gazettes", 5);
```
Comment (Rushi98) [inline app/src/main/java/com/csatimes/dojma/activities/MainActivity.java]: Use defined constants

File: app/src/main/res/color/bottom_navigation_colors.xml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <item
+        android:state_checked="true"
+        android:color="@color/grey100" />
+    <item
+        android:state_checked="false"
+        android:color="@color/grey400" />
+</selector>
```
Comment (Rushi98) [inline app/src/main/res/color/bottom_navigation_colors.xml]: Newline at end of each file

File: app/src/main/res/layout/fragment_favourites.xml [hunk 5]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:orientation="vertical" android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+</LinearLayout>
```
Comment (Rushi98) [inline app/src/main/res/layout/fragment_favourites.xml]: Newline

File: app/src/main/java/com/csatimes/dojma/fragments/favouritesfragment.java [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+import com.csatimes.dojma.R;
+
+import androidx.annotation.NonNull;
+import androidx.fragment.app.Fragment;
+
+public class favouritesfragment extends Fragment {
+
+    public favouritesfragment() {
+        // Required empty public constructor
```
Comment (Rushi98) [inline app/src/main/java/com/csatimes/dojma/fragments/favouritesfragment.java]: Use singleton style method for fragment creation (the `public static Fragment newInstance(Bundle args){}` method)

**Review by Rushi98 — CHANGES_REQUESTED** [review]

**Comment by Rushi98** [comment]
See the changes. I haven't tested it yet, but seems ok.

File: .idea/inspectionProfiles/Project_Default.xml (hunk 1 above)
Comment (Agrim1234) [reply .idea/inspectionProfiles/Project_Default.xml]: I have'nt deleted them pata nahi kaha se u gayi

Comment (Agrim1234) [reply .idea/inspectionProfiles/Project_Default.xml]: Same for the other file

**Comment by Agrim1234** [comment]
Rest all are done

**Comment by Rushi98** [comment]
@tanmaydixit please review this pr

**Pull request merged**